spec/controllers/api/bot_auth_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Api::SchedulesController do
+  include Devise::TestHelpers
+
+  describe 'Bot authentication' do
+
+    let(:device) { FactoryGirl.create(:device) }
+
+    it 'authorizes using MeshBlu UUID / Token' do
+      @request.headers["bot_token"] = device.token
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.